package-lock.json

```diff
@@ -16,10 +16,10 @@
       "integrity": "sha1-c7Xuyj+rZT49P5Qis0GtQiBdyWU=",
```

Comment:
Is there anything in this lock file that is needed for this PR? If not, I would rather see this not committed. To revert you can easily recover the version in the master branch with `git checkout master -- package-lock.json`

Reply:
Sure, will get it out.

Reply:
And thanks for the git command - that's useful to know!